xrpl/utils/tx_parser/utils.py

```diff
@@ -0,0 +1,1052 @@
+"""Utils for balance_changes and orderbook_changes."""
+from __future__ import annotations
+
+from datetime import datetime
+from decimal import Decimal
+from typing import Any, Callable, Dict, List, Optional, Tuple, Type, Union
+
+from pydash import compact, filter_, flatten, group_by, map_, map_values  # type: ignore
+from typing_extensions import Literal
+
+from xrpl.constants import XRPLException
+from xrpl.utils.xrp_conversions import drops_to_xrp
+
+"""
+###########################################
```

Comment:
This file is very long, and it looks like it has distinct parts. Why is it not split into multiple files?